modules/voting/funcs/main.php

```diff
@@ -168,19 +168,33 @@
 
         $acceptcm = (int) $row['acceptcm'];
         $logfile = md5(NV_LANG_DATA . $global_config['sitekey'] . $client_info['ip'] . $vid) . '.' . $log_fileext;
-
-        if (file_exists($dir . '/' . $logfile)) {
-            $timeout = filemtime($dir . '/' . $logfile);
-            $timeout = ceil(($difftimeout - NV_CURRENTTIME + $vtime) / 60);
-            $note = sprintf($lang_module['timeoutmsg'], $timeout);
-        } elseif ($count <= $acceptcm) {
-            $in = implode(',', $array_id);
-            $sql = 'UPDATE ' . NV_PREFIXLANG . '_' . $module_data . '_rows SET hitstotal = hitstotal+1 WHERE vid =' . $vid . ' AND id IN (' . $in . ')';
-            $db->query($sql);
-            file_put_contents($dir . '/' . $logfile, '', LOCK_EX);
-            $note = $lang_module['okmsg'];
+        if (defined('NV_IS_USER') && ($row['vote_one'] == 1)) {
```

Comment:
@baonv-dev thống nhất dùng `and` và `or` thay cho `&&` và `||`.

Reply:
Với lại dùng `($row['vote_one'] == 1)` là thừa đó em, để `$row['vote_one'] == 1` là đủ rồi. Hoặc để `!empty($row['vote_one'])` sẽ ok hơn.